src/main/java/org/dependencytrack/parser/hyades/NotificationModelConverter.java

```diff
@@ -293,6 +303,36 @@ private static Project convert(final org.dependencytrack.model.Project project)
         return builder.build();
     }
 
+    private static ComponentAnalysisCompleteSubject convert(ComponentAnalysisComplete componentAnalysisComplete) {
```

Comment:
There should be a test for this in [`NotificationModelConverterTest`](https://github.com/DependencyTrack/hyades-apiserver/blob/main/src/test/java/org/dependencytrack/parser/hyades/NotificationModelConverterTest.java).